src/dialogue_helpers.h

```diff
@@ -91,7 +91,7 @@ struct talk_effect_fun_t {
         void set_npc_goal( const JsonObject &jo, std::string_view member, bool is_npc = false );
         void set_destination( const JsonObject &jo, const std::string &member, bool is_npc = false );
         void set_revert_location( const JsonObject &jo, const std::string_view &member );
-        void set_guard_pos( const JsonObject &jo, const std::string &member, bool is_npc = false );
+        void set_guard_pos( const JsonObject &jo, const std::string_view &member, bool is_npc = false );
```

Comment:
`string_view` should (generally) not be passed by const reference.

I guess I'm going to need to go write a check for this too... :)

Reply:
Welp, I'll leave it as is for now but yeah handholding check would be nice 😊